### DISCUSSION TITLE
Powering ESP32S -WROVER

### DISCUSSION THREAD

**Discussion by Turboen** [bitcraze/discussions#634] [Q&A]
Hey,
I was wondering if it was possible to power the ESP32S WROVER through the 3.3v pin?
I am looking to do some operations on the MCU, that is capable of transmitting data through wifi to my computer while the drone is operating.
However, as I think i understand, the MCU sometimes draws more current that 100 mA when using Wifi. Any way to do this? Or is there any way to utilize the onboard MCU of the crazyflie.
What i need is to have the data from multiple photodiodes be sent to my computer, such that i can supply the crazyflie with different commands in response to the received data.

**Comment by ataffanel** [comment]
Hi,

I would suggest you connect a 3V voltage regulator to VCOM and then power the ESP32 from your own regulated supply. VCOM is supplied directly from the battery (or 5V USB when the usb cable is connected), so you will not have problem with the ESP32 power draw there.

If you really would like to use VCC, you can try but it is indeed possible that you will hit the current limit. You might try to mitigate the problem by running the ESP32 slow and with low radio output power. But the best would really be to have your own regulator, this way power will not be a worry.
